tools/idf_monitor.py

```diff
@@ -159,7 +179,9 @@ def run(self):
             self.console.cleanup()
 
     def _cancel(self):
-        if os.name == 'posix':
+        if self.wsl:
+            pass
```

Comment:
What do you think about `if os.name == 'posix' and not self.is_wsl` ?

Reply:
I would prefer to keep the WSL workaround separate from the original implementation, as the original will not introduce extra latency when typing on the console (once stdin from console will be implemented ...).

Reply:
Uh, sorry, I misunderstood your suggestion.
I hope I got it now. I have slightly reworked the variable naming, changed the condition like you suggested and already squashed/rebased the commit.